awscli/customizations/s3/s3handler.py

```diff
@@ -186,7 +187,8 @@ def _enqueue_tasks(self, files):
                                          message=warning_message)
                 self.result_queue.put(warning)
             # Warn and skip over glacier incompatible tasks.
-            elif not filename.is_glacier_compatible():
+            elif not filename.is_glacier_compatible(
```

Comment:
Why couldn't we just make the logic:
```py
elif not self.params.get('force_glacier_transfer') and not filename.is_glacier_compatible():
```
instead of adding it as a parameter to the ``is_glacier_compatible()`` method.